### PR TITLE
Add new resource, fixes, refactoring, add new features in README.md

### PR DESCRIPTION
**Pull request by AmsTaFFix**

### CODE REVIEW

File: src/Enum/RemovedSubscribersGetRequest/Option.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * @author Nikita Sapogov <sapogov.n@citilink.ru>
+ */
+class Option extends Enum
+{
+    /**
+     * Return all subscriber properties and some general information
+     */
+    const CUSTOMS = 'Customs';
+}
```
Comment (liderman) [inline src/Enum/RemovedSubscribersGetRequest/Option.php]: Add an empty line to the end

File: src/Enum/RemovedSubscribersGetRequest/RemoveType.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    /**
+     * Removed because email server returned User Unknown bounce (email address does not exist)
+     */
+    const USER_UNKNOWN = 'UserUnknown';
+
+    /**
+     * Removed using API
+     */
+    const API = 'Api';
+}
```
Comment (liderman) [inline src/Enum/RemovedSubscribersGetRequest/RemoveType.php]: Add an empty line to the end

File: src/Model/RemovedSubscribersGetResponse/RemovedSubscriber.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+     * If you want get subscriber's data, you should set option="Customs" in
+     * {@see RemovedSubscribersGetRequest}
+     *
+     * @return SubscriberData|null Subscriber's data
+     */
+    public function getSubscriberData(): ?SubscriberData
+    {
+        return $this->subscriberData;
+    }
+}
```
Comment (liderman) [inline src/Model/RemovedSubscribersGetResponse/RemovedSubscriber.php]: Add an empty line to the end

File: src/Model/SubscriberData.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    /**
+     * Get properties
+     *
+     * @return SubscriberProperty[]|iterable Properties
+     */
+    public function getProperties(): iterable
+    {
+        return $this->properties;
+    }
+}
```
Comment (liderman) [inline src/Model/SubscriberData.php]: Add an empty line to the end

File: src/Request/RemovedSubscriberGetRequest.php [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getUri(): string
+    {
+        return '/Api/RemovedSubscribers';
+    }
+}
```
Comment (liderman) [inline src/Request/RemovedSubscriberGetRequest.php]: Add an empty line to the end

File: src/RequestSenderInterface.php [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+{
+    /**
+     * Send request
+     *
+     * @param RequestInterface $request Request
+     *
+     * @return ResponseInterface Response
+     */
+    public function send(RequestInterface $request): ResponseInterface;
+}
```
Comment (liderman) [inline src/RequestSenderInterface.php]: Add an empty line to the end

File: src/Resource/RemovedSubscribersResource.php [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        \DateTime $endDate = null,
+        Option $option = null
+    ): RemovedSubscribersGetResponse {
+        $response = $this->requestSender->send(
+            new RemovedSubscriberGetRequest($listIds, $removeTypes, $startDate, $endDate, $option)
+        );
+
+        return new RemovedSubscribersGetResponse($response);
+    }
+}
```
Comment (liderman) [inline src/Resource/RemovedSubscribersResource.php]: Add an empty line to the end

File: src/Response/RemovedSubscribersGetResponse.php [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+            yield new RemovedSubscriber(
+                strval($node->Email),
+                intval($node->ListId),
+                new \DateTime(strval($node->UnsubscribedOn)),
+                $subscriberData
+            );
+        }
+    }
+}
```
Comment (liderman) [inline src/Response/RemovedSubscribersGetResponse.php]: Add an empty line to the end

File: src/SubscriberDataParser.php [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+                            'Unable recognize type and create value for subscriber\'s property. XML: [%s]',
+                            $node->asXML()
+                        )
+                    );
+            }
+
+            yield new SubscriberProperty($id, $type, $friendlyName, $name, $description, $source, $value);
+        }
+    }
+}
```
Comment (liderman) [inline src/SubscriberDataParser.php]: Add an empty line to the end

Comment (AmsTaFFix) [inline src/SubscriberDataParser.php]: fixed

File: src/Response/RemovedSubscribersGetResponse.php (hunk 8 above)
Comment (AmsTaFFix) [inline src/Response/RemovedSubscribersGetResponse.php]: fixed

File: src/Resource/RemovedSubscribersResource.php (hunk 7 above)
Comment (AmsTaFFix) [inline src/Resource/RemovedSubscribersResource.php]: fixed

File: src/RequestSenderInterface.php (hunk 6 above)
Comment (AmsTaFFix) [inline src/RequestSenderInterface.php]: fixed

File: src/Request/RemovedSubscriberGetRequest.php (hunk 5 above)
Comment (AmsTaFFix) [inline src/Request/RemovedSubscriberGetRequest.php]: fixed

File: src/Model/SubscriberData.php (hunk 4 above)
Comment (AmsTaFFix) [inline src/Model/SubscriberData.php]: fixed

File: src/Model/RemovedSubscribersGetResponse/RemovedSubscriber.php (hunk 3 above)
Comment (AmsTaFFix) [inline src/Model/RemovedSubscribersGetResponse/RemovedSubscriber.php]: fixed

File: src/Enum/RemovedSubscribersGetRequest/RemoveType.php (hunk 2 above)
Comment (AmsTaFFix) [inline src/Enum/RemovedSubscribersGetRequest/RemoveType.php]: fixed

File: src/Enum/RemovedSubscribersGetRequest/Option.php (hunk 1 above)
Comment (AmsTaFFix) [inline src/Enum/RemovedSubscribersGetRequest/Option.php]: fixed

**Pull request merged**